addon/globalPlugins/remoteClient/local_machine.py

```diff
@@ -11,17 +11,17 @@
 import logging
 logger = logging.getLogger('local_machine')
 
-class LocalMachine(object):
+class LocalMachine:
 
 	def __init__(self):
 		self.is_muted = False
 		self.receiving_braille=False
-		
-	def play_wave(self, fileName, async, **kwargs):
+
+	def play_wave(self, fileName, asynchronous=True, **kwargs):
 		if self.is_muted:
 			return
 		if os.path.exists(fileName):

```

Comment:
I think we should add something like this to add version 2.2 compat

```suggestion
		asynchronous = kwargs.pop("async", asynchronous)
		if os.path.exists(fileName):
```

Reply:
Can you explain the case you are thinking of a little more?

Reply:
Oh, I understand. Because it is registered as a callback handler for "msg_wave" in `MasterSession.__init__()`  and `TCPTransport.parse()` just calls each callback with whatever args were captured on the other machine. So on NVDARemote 2.2 which has `async` argument will end up making this call `play_wave` with an argument it doesn't handle.

Reply:
Though, to be honest, I'm not sure that the remote machine should be controlling "how" the wave is played at all. I'm tempted to rely on the defaults for `play_wave` instead.

Reply:
For now I have tried to make it bidirectionally compatible. Still interested to hear what people think about squashing the asynchronous argument entirely.

Reply:
It makes sense to squash asynchronous, actually. We don't want to make wave playback on the controlled machine block the main thread of the controlling machine.